packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/SQLEditor.tsx

```diff
@@ -1,46 +1,83 @@
+"use client";
+
 import { PlayIcon } from "lucide-react";
 import { useQueryState } from "nuqs";
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
 import { useForm } from "react-hook-form";
+import { Table } from "@latticexyz/config";
+import Editor from "@monaco-editor/react";
 import { Button } from "../../../../../../components/ui/Button";
-import { Form, FormControl, FormField, FormItem } from "../../../../../../components/ui/Form";
-import { Input } from "../../../../../../components/ui/Input";
+import { Form, FormField } from "../../../../../../components/ui/Form";
+import { cn } from "../../../../../../utils";
+import { monacoOptions } from "./consts";
+import { useMonacoSuggestions } from "./useMonacoSuggestions";
+import { useQueryValidator } from "./useQueryValidator";
+
+type Props = {
+  table?: Table;
```

Comment:
does this just aid in autocomplete/validation?

Reply:
indeed, only need columns and table name from it atm. I suppose it would be somewhat cleaner to just pass these as props explicitly? I remember receiving a similar suggestion in the past.